core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -95,6 +96,7 @@ void writeFrame(
 
   private final TransportTracer transportTracer;
   private final Framer framer;
+  private final CallOptions callOptions;
```

Comment:
In some ways I'd prefer if we didn't store the callOptions and instead pulled out only the pieces we need (in this case, shouldBeCountedForInUse). Otherwise it gets more confusing as to why the callOptions are there and when should the be consulted.